tests/test_tiatoolbox.py

```diff
@@ -6,8 +6,11 @@
 from tiatoolbox.dataloader.slide_info import slide_info
 from tiatoolbox.dataloader.save_tiles import save_tiles
 from tiatoolbox.dataloader import wsireader
+from tiatoolbox.tools.stainnorm import reinhard_colour_normaliser
+from tiatoolbox.tools.stainnorm import stain_normaliser
```

Comment:
Please use 'from tiatoolbox.tools.stainnorm import reinhard_colour_normaliser, stain_normaliser` instead of importing in two lines.

Reply:
Fixed in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/46b2c26c5dc5862f1e4106f2a5347ddb54a59d46)